tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+      "baseUrl": ".",
+      "paths": {
+          "@src/*": ["src/*"]
+      },
+      "target": "es6",
+      "module": "commonjs",
+      "moduleResolution": "node",
+      "rootDir": "./src",
+      "outDir": "dist/scripts",
+      "strict": true,
+      "sourceMap": true,
+      "jsx": "react",
+      "lib": [
+        "DOM",
+        "DOM.Iterable"
```

Comment:
?

Reply:
A lot of this came from a template repo I started with. It was typescript+react. I will remove all extraneous options.

Reply:
DOM is required (I think) for DOM definitions - `window`, `document`, etc. in TS files.